src/components/contact-govuk-one-login/index.njk

```diff
@@ -134,5 +135,14 @@
     src="{{ webchatSource }}" 
     data-company="hgsgds" data-brand="hgsgds">
   </script>
+  <script type="text/javascript" nonce='{{scriptNonce}}'>
+    var launchWebchatButton = document.querySelector("[data-launch-webchat]")
```

Comment:
I'm not entirely sure why our eslint rules are not being applied on save but I think we should have semicolons at the end of this line and the ones inside the if.